tests/timeline.py

```diff
@@ -0,0 +1,148 @@
+from datetime import datetime, timedelta, date
+
+from ics import Calendar, Event
+from ics.timezone import UTC
+
+
+def make_calendar():
```

Comment:
Could you make this a [pytest fixture](https://docs.pytest.org/en/6.2.x/fixture.html) please ?

Reply:
👍🏼